src/common/windows/pe_util.cc

```diff
@@ -280,7 +280,12 @@ bool PrintPEFrameData(const wstring & pe_file, FILE * out_file)
         exception_rva,
         &img->LastRvaSection));
   for (DWORD i = 0; i < exception_size / sizeof(*funcs); i++) {
-    DWORD unwind_rva = funcs[i].UnwindInfoAddress;
+    DWORD unwind_rva;
+#if defined(_M_ARM64) || defined(__aarch64__)
+    unwind_rva = funcs[i].UnwindData;
+#else
+    unwind_rva = funcs[i].UnwindInfoAddress;
+#endif
```

Comment:
This would make a less change and unlikely conflicts with future upstream updates,
```suggestion
#if defined(_M_ARM64) || defined(__aarch64__)
    DWORD unwind_rva unwind_rva = funcs[i].UnwindData;
#else
    DWORD unwind_rva unwind_rva = funcs[i].UnwindInfoAddress;
#endif
```